pkg/local_object_storage/metabase/version.go

```diff
@@ -0,0 +1,41 @@
+package meta
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"go.etcd.io/bbolt"
+)
+
+// Version contains current metabase version.
+const Version = 0
```

Comment:
In which packages this is going to be used?

Reply:
Fixed